draft-quic-atsss-reqs.mkd

```diff
@@ -138,7 +138,7 @@ informative:
     
 --- abstract
 
-This document briefly presents the Access Traffic Steering, Switching, and Splitting (ATSSS) service being specified within the 3rd Generation Partnership Project (3GPP). 
+This document briefly presents the Access Traffic Steering, Switching, and Splitting (ATSSS) service being specified within the 3rd Generation Partnership Project (3GPP). ATSSS provides network support for multihomed devices select a path for transmission (steer), move traffic from one path to another (switch), or use multiple path simutaneously (split). TS 23.501 specifies an ATSSS for TCP traffic only. This document presents a snap-shot of the on-going discussion in 3GPP to enable ATSSS for non-TCP traffic, based on the use of QUIC.
```

Comment:
Minor nits. Looks good!
```suggestion
This document briefly presents the Access Traffic Steering, Switching, and Splitting (ATSSS) service being specified within the 3rd Generation Partnership Project (3GPP). ATSSS provides network support for multihomed devices to select a path for transmission (steer), move traffic from one path to another (switch), or use multiple paths simutaneously (split). TS 23.501 specifies an ATSSS for TCP traffic only. This document presents a snap-shot of the on-going discussion in 3GPP to enable ATSSS for non-TCP traffic, based on the use of QUIC.
```

Reply:
Github doesn't highlight my edits properly:
- s/multihomed devices select/multihomed devices to select/
- s/use multiple path simutaneously/use multiple paths simutaneously/